config/crawler-user-agents.json

```diff
@@ -1,6 +1,6 @@
 [
   {
-    "pattern": "",
+    "pattern": "^\s*$",
```

Comment:
Let's add a test case to bad replicants for a user agent with just spaces.

Reply:
Test added in the previous commit: https://github.com/socrata-platform/Voight-Kampff/blob/91523a9522181d25964efa7e6df62b3aefb4711d/spec/support/bad_replicants.rb#L5

Lemme check if it works. Rspec not cooperating with me at the moment

Reply:
and a good bot that is not just spaces but contains a space.